src/main/java/io/opentracing/contrib/spanmanager/tracer/ManagedSpanBuilder.java

```diff
@@ -101,8 +100,14 @@ public SpanBuilder withStartTimestamp(long microseconds) {
         return rewrap(delegate.withStartTimestamp(microseconds));
     }
 
+    /**
+     * @return Empty set of baggage items.
+     * @deprecated Don't call this anymore, SpanBuilder and SpanContext are separated since opentracing-api v0.22.0.
+     */
     public Iterable<Map.Entry<String, String>> baggageItems() {
```

Comment:
Probably ok to leave for now as would enable tracing to proceed, would just impact apps expecting baggage, which is probably a smaller subset of apps.